packages/website/ts/components/wallet/wallet.tsx

```diff
@@ -444,16 +418,19 @@ export class Wallet extends React.Component<WalletProps, WalletState> {
     private _renderAccessoryItems(config: AccessoryItemConfig): React.ReactElement<{}> {
         const shouldShowWrappedEtherAction = !_.isUndefined(config.wrappedEtherDirection);
         const shouldShowToggle = !_.isUndefined(config.allowanceToggleConfig);
+        // if we don't have a toggle, we still want some space to the right of the "wrap" button so that it aligns with
+        // the "unwrap" button in the row below
+        const toggle = shouldShowToggle ? (
+            this._renderAllowanceToggle(config.allowanceToggleConfig)
+        ) : (
+            <div style={{ width: NO_ALLOWANCE_TOGGLE_SPACE_WIDTH }} />
```

Comment:
Maybe you can achieve this layout by using justify-content: space-between (and removing the fixed width), which would push the elements far apart -- so the balance and toggle will be in the same place regardless of whether there is an element in between them. For the alignment of the wrap button, you may have to add some margin/spacing on the left to make it look like it does now with this approach.

Reply:
Alright I looked into this a bit more and I think it might be ok to keep this as is for now since it is a pretty special case and we will still need some magic numbers to get the "wrap" button to align with the "unwrap" button". `AllowanceToggle` is not set up to show undefined configurations so it will require a refactor of that component

Reply:
Just occurred to me that it may have made sense to use a column layout. This works though -- a consideration for the future.
http://basscss.com/#basscss-grid